### DISCUSSION TITLE
I can't complete my project on cloud.walletconnect.com

### DISCUSSION THREAD

**Discussion by ngn999** [WalletConnect/walletconnect-monorepo#5235] [Developer Support]
My project id is 1767a1f4754e474309aea0afcb51cab2.
It's said, complete your project: 1/2 completed. 
Yes, I had verifed my domain. but what is the second quest?

<img width="622" alt="image" src="https://github.com/user-attachments/assets/bd9917fe-5429-4e98-994e-99233e203065">

**Comment by glitch-txs** [comment]
hello, we recently pushed some fixed on Cloud, could you check again?
